requirements.txt requiring a specific minor version prevents pip install

I get this error when running pip install dydx-v3-python, on version 3.11, on a macbook pro m1: 

Building wheels for collected packages: aiohttp
  Building wheel for aiohttp (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for aiohttp (pyproject.toml) did not run successfully.
  │ exit code: 1

      aiohttp/_websocket.c:198:12: fatal error: 'longintrepr.h' file not found
        #include "longintrepr.h"
                 ^~~~~~~~~~~~~~~
      1 error generated.
      error: command '/usr/bin/clang' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for aiohttp
Failed to build aiohttp
ERROR: Could not build wheels for aiohttp, which is required to install pyproject.toml-based projects



I looked up this issue and found: https://github.com/aio-libs/aiohttp/issues/6600

"Fails to build on Python 3.11 - longintrepr.h: No such file or directory"

some of the comments include:

"I don't believe this is happening with aiohttp 3.8.3+. You must update for Python 3.11+ support. The C-extension files in old sdists are generated with an older Cython version, there is no way to retroactively substitute those sdists on PyPI.3.8.3 solved the problem for us. Apologies."

Then when I looked in the requirements.txt file here, I found:

aiohttp==3.8.1

That's why it's failing. Can we update the requirements.txt to be aiohttp>=3.8.1 or similar? I want to install a later version of aiohttp to make this work

Thanks!

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -2,7 +2,7 @@ version: 2
 jobs:
   test:
     docker:
-      - image: circleci/python:3.11
+      - image: circleci/python:3.9
       - image: trufflesuite/ganache-cli:v6.6.0
         command: -d -k=petersburg -i 1001
     working_directory: ~/build
@@ -16,7 +16,7 @@ jobs:
           command: tox
   lint:
     docker:
-      - image: circleci/python:3.11
+      - image: circleci/python:3.9
     working_directory: ~/build
     steps:
       - checkout
@@ -28,7 +28,7 @@ jobs:
           command: flake8
   publish:
     docker:
-      - image: circleci/python:3.11
+      - image: circleci/python:3.9
     working_directory: ~/build
     steps:
       - checkout

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -2,7 +2,7 @@ version: 2
 jobs:
   test:
     docker:
-      - image: circleci/python:3.6.4
+      - image: circleci/python:3.11
       - image: trufflesuite/ganache-cli:v6.6.0
         command: -d -k=petersburg -i 1001
     working_directory: ~/build
@@ -16,7 +16,7 @@ jobs:
           command: tox
   lint:
     docker:
-      - image: circleci/python:3.6.4
+      - image: circleci/python:3.11
     working_directory: ~/build
     steps:
       - checkout
@@ -28,7 +28,7 @@ jobs:
           command: flake8
   publish:
     docker:
-      - image: circleci/python:3.6.4
+      - image: circleci/python:3.11
     working_directory: ~/build
     steps:
       - checkout

--- a/README.md
+++ b/README.md
@@ -15,7 +15,7 @@
 
 Python client for dYdX (v3 API).
 
-The library is currently tested against Python versions 2.7, 3.4, 3.5, and 3.6
+The library is currently tested against Python versions 2.7, 3.4, 3.5, 3.6, 3.9, and 3.11.
 
 ## Installation
 

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,7 +1,7 @@
-aiohttp==3.8.1
+aiohttp>=3.8.1
 cytoolz==0.11.2
 dateparser==1.0.0
-ecdsa==0.16.0
+ecdsa>=0.16.0
 eth_keys
 eth-account>=0.4.0,<0.6.0
 mpmath==1.0.0

--- a/setup.py
+++ b/setup.py
@@ -3,10 +3,10 @@ from setuptools import setup, find_packages
 LONG_DESCRIPTION = open('README.md', 'r').read()
 
 REQUIREMENTS = [
-    'aiohttp==3.8.1',
+    'aiohttp>=3.8.1',
     'cytoolz==0.11.2',
     'dateparser==1.0.0',
-    'ecdsa==0.16.0',
+    'ecdsa>=0.16.0',
     'eth_keys',
     'eth-account>=0.4.0,<0.6.0',
     'mpmath==1.0.0',

--- a/setup.py
+++ b/setup.py
@@ -21,7 +21,7 @@ REQUIREMENTS = [
 
 setup(
     name='dydx-v3-python',
-    version='1.9.1',
+    version='1.9.2',
     packages=find_packages(),
     package_data={
         'dydx3': [
@@ -48,6 +48,8 @@ setup(
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.11',
         'Programming Language :: Python',
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],

--- a/tox.ini
+++ b/tox.ini
@@ -3,7 +3,7 @@ exclude = dydx3/starkex/starkex_resources
 per-file-ignores = __init__.py:F401
 
 [tox]
-envlist = python2.7, python3.4, python3.5, python3.6
+envlist = python2.7, python3.4, python3.5, python3.6, python3.9, python3.11
 
 [testenv]
 commands =

